e2echart/e2e-chart-template.html

```diff
@@ -487,6 +495,9 @@ <h5 class="modal-title">Resource</h5>
         timelineGroups.push({group: "api-unreachable", data: []})
         createTimelineData(isAPIUnreachableFromClientValue, timelineGroups[timelineGroups.length - 1].data, eventIntervals, isAPIUnreachableFromClientActivity, regex)
 
+        timelineGroups.push({group: "staticpod-install", data: []})
+        createTimelineData(isStaticPodInstallMonitorValue, timelineGroups[timelineGroups.length - 1].data, eventIntervals, isStaticPodInstallMonitorActivity, regex)
+
```

Comment:
Are the solid blue bars that look to be overlapping with a darker shade of blue what you would expect here: https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/29462/pull-ci-openshift-origin-master-e2e-gcp-ovn-rt-upgrade/1881869312688394240

Almost looks like locators are overlapping, but also is it supposed to show overlap for all that time?

Reply:
no, this is from a run with an earlier version of this PR, I have fixed the issue, any run from today should not have the large blue bar.